bin/minifollowups/pycbc_page_coincinfo

```diff
@@ -46,6 +47,9 @@ trig_input.add_argument('--trigger-id', type=int,
 parser.add_argument('--include-summary-page-link', action='store_true',
     help="If given, will include a link to the DQ summary page on the "
          "single detector trigger tables.")
+parser.add_argument('--include-gracedb-link', action='store_true',
+    help="If given, will provide a link to search GraceDB for events "
+         "within a 3s window around the coincidence time.")
```

Comment:
I see the two options `--include-summary-page-link` and `--include-gracedb-link` are used by two scripts. Have you considered using a `add_followup_options()` function to the `followup` module?